RUMM-3470: Update documentation for Android SDK v2 (#19062)
DataDog/documentation: content/en/real_user_monitoring/android/integrated_libraries.md

If you use Jetpack Compose in your application, take a look at Datadog's [dedicated Jetpack Compose library][7].

### RxJava

> If you use RxJava in your application, take a look at Datadog's [dedicated RxJava library][8].

Reviewer: ```suggestion
If you use RxJava in your application, see Datadog's [dedicated RxJava library][8].
```

### Picasso
 

> If you use Picasso, let it use your `OkHttpClient` instrumented with Datadog SDK for RUM and APM information about network requests made by Picasso.

Reviewer: ```suggestion
If you use Picasso, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about network requests made by Picasso.
```

> If you use Picasso, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about network requests made by Picasso.

Reviewer: ```suggestion
If you use Picasso, use it with the `OkHttpClient` that's been instrumented with the Datadog SDK for RUM and APM information about network requests made by Picasso.
```

 
### Retrofit
 

> If you use Retrofit, let it use your `OkHttpClient` instrumented with Datadog SDK for RUM and APM information about network requests made with Retrofit.

Reviewer: ```suggestion
If you use Retrofit, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about network requests made with Retrofit.
```

> If you use Retrofit, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about network requests made with Retrofit.

Reviewer: ```suggestion
If you use Retrofit, use it with the `OkHttpClient` that's been instrumented with the Datadog SDK for RUM and APM information about network requests made with Retrofit.
```

 
### SQLDelight
 

> If you use SQLDelight in your application, take a look at Datadog's [dedicated SQLDelight library][4].

Reviewer: ```suggestion
If you use SQLDelight in your application, see Datadog's [dedicated SQLDelight library][4].
```

 
### Apollo (GraphQL)
 

> If you use Apollo, let it use your `OkHttpClient` instrumented with Datadog SDK for RUM and APM information about all the queries performed through Apollo client.

Reviewer: ```suggestion
If you use Apollo, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about all the queries performed through Apollo client.
```

> If you use Apollo, use it with the `OkHttpClient` that's been instrumented with Datadog SDK for RUM and APM information about all the queries performed through Apollo client.

Reviewer: ```suggestion
If you use Apollo, use it with the `OkHttpClient` that's been instrumented with the Datadog SDK for RUM and APM information about all the queries performed through Apollo client.
```
